README.md

```diff
@@ -160,14 +164,47 @@ Note: Don't use `@` symbol when adding username.
 - [UnDraw](https://undraw.co/illustrations)
 
 ## Deployment 📦 
-Once you have done with your setup. You need to put your website online!
-I highly recommend to use [Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) to achieve this on the EASIEST WAY. 
+Once you have done with your setup. You need to put your website Online!
+We highly recommend to use [Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) to achieve this on the EASIEST WAY. Below is a brief on how to.
 
-You could also deploy it directly with Netlify by linking your own repo.
+#### Deploying to Github Pages
 
-[![Deploy To Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/saadpasta/developerFolio)
+This section guides you to deploying this Repository to Github pages.
+
+- First, follow the steps [here](#getting-started-) to clone the Repository to your Computer and Install the dependencies. You should clone you fork and not the original Repository.
+
+- Next, Edit  `src/portfolio.js` to make the site as your *own*.
+
+- Then, navigate to `package.json` and enter your domain name instead of `https://developerfolio.js.org/` in `homepage` variable. For example, if you want your site to be `https://<your-username>.github.io/developerFolio`, add the same to the homepage section of `package.json`. In short you can also add `/devloperFolio` to `package.json` as both are exactly same. Upon doing so, you say `create-react-app` to add the path assets accordingly. If you going to host in the root of domain you can very well remove the `homepage` variable as by default it assumes the you host the site in the root of the domain.  `https://<your-username>.github.io` is considered as root of the domain while `https://<your-username>.github.io/developerFolio` is not.
```

Comment:
Break this into paragraphs I think and change "you say create react app" to "you tell create react app"